src/audio/host.c

```diff
@@ -383,6 +383,13 @@ static int host_copy_normal(struct comp_dev *dev)
 		return ret;
 	}
 
+	if (hd->chan->status != COMP_STATE_ACTIVE) {
+		ret = dma_start(hd->chan);
```

Comment:
@keyonjie the commit message doesnt explain why this change is needed? Isnt the dma started already during the START trigger?

Reply:
oops, this was kept by mistake from the old version, it is not needed anymore, let me update it.